groceries_app/static/images/linkedin.svg

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="iso-8859-1"?>

```

Comment:
Can you not use a standard image fromat like png/jpeg?
This kinda feels unecessary.

Reply:
Agreed. There can be benefits to using svgs, could be useful for using vector graphics but this is probably not a situation where it matters.

Something I found: "If you know you need vector art, SVG is the choice. Vector art is visually crisp and tends to be a smaller file size than raster graphics like JPG. If you care a lot about the flexibility and responsiveness of the graphic, SVG is the way."

Reply:
Oh in that case svg does seem to be the better choice.